types/list.go

```diff
@@ -156,8 +157,6 @@ type ObjectInfo struct {
 	SourceType storageType.SourceType `json:"source_type"`
 	// checksums define the root hash of the pieces which stored in a SP.
 	Checksums [][]byte `json:"checksums" traits:"omit"`
-	// secondary_sp_addresses define the addresses of secondary_sps
-	SecondarySpAddresses []string `json:"secondary_sp_addresses"`
```

Comment:
Please submit this PR to https://github.com/bnb-chain/greenfield-go-sdk/pull/98.